### DISCUSSION TITLE
Is there a way to save responses of multiple requests in a local folder

### DISCUSSION THREAD

**Discussion by Sateswar** [thunderclient/thunder-client-support#1060] [Q&A] [answered]
I have multiple requests and generally use "Run All" options to see the response. However, I would like to save all the response JSONs in a folder so that I can compare Last Response vs current response. 

Thank you in advance.

**Comment by rangav** [comment, marked as answer]
Hi @Sateswar  Currently not possible in easy way.

Please follow below steps
1. The data is stored here - https://github.com/rangav/thunder-client-support#storage
2. The Response metadata is saved in `thunderResponses.json` file
3. The Actual response body is saved in files in `res-files` folder
4. You have to match them with request id
5. Copy the `json` file and `res-files` folder to some other backup folder
6. When VSCode restarts these files are cleared automatically.

Hope the above is useful

**Reply by Sateswar** [reply]
Thank @rangav for the quick response! You are awesome.
